app/src/main/res/values-nb-rNO/strings_help.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Help -->
+    <string name="help_title">Hjelp</string>
+    <string name="help_navigation_title">Navigasjon</string>
+    <string name="help_navigation_text">
+        Dra oppover fra hjemmeskjermen for å vise alle programmer, og ned for å gå tilbake.
+        \n
+        \nDra nedover fra hjemmeskjermen for å vise favoritter, og opp for å gå tilbake.
+        \n
+        \nTrykk på menyknappen (når du viser favoritter) eller oppstarterikonet for å åpne menyen.
+        \n
+        \nKlikk-hold et program eller en mappe for å vise respektive innstillinger.
+    </string>
+    <string name="help_notification_title">Merknad</string>
+    <string name="help_notification_text">
+        Viser et oppsprett av favoritter over andre programmer når du forlater hjemmeskjermen.
+        </string>
+    <string name="help_list_update_title">Listegjenoppfriskning</string>
+    <string name="help_list_update_text">
+        En oppsprettsmelding vises ved gjenoppfrisking av programlisten og ikoner,
+        når et program legges til eller fjernes, og når ikonpakken endres).
+    </string>
+    <string name="help_folders_title">Mapper</string>
+    <string name="help_folders_text">
+        Håndter mapper eller favorittmerk dem med menyknappen, eller klikk på tittelfeltet
+        til en åpen mappe for å opprette, gi nytt navn, eller putte programmer i den.
+        \nFjerning av mapper putter alle programmer i dem tilbake i hovedlisten.
+    </string>
+    <string name="help_search_title">Søk</string>
+    <string name="help_search_text">
+        Finn ethvert program med søkeikonet. Det kan også favorittmerkes for å være tilgjengelig
+        fra merknaden, eller skjules.
+        \n
+        \nTrykk \"Enter\" for å åpne hvis det er kun ett søkeresultat.
+    </string>
+
+    <!-- Changelog -->
+    <string name="changelog_title">Endringslogg</string>
+    <string name="changelog_mentions">\"@pseudo\" indikerer brukeren som foreslo idéen eller rapporterte feilen</string>
+    <string name="changelog_folder">changelog-nb</string>
```

Comment:
```suggestion
    <string name="changelog_folder">changelog-en</string>
```
This should point towards the English changelog (unless French is more appropriate in Norway?), otherwise it will only display the message from the "error_changelog_folder" string.

For info:
I included the changelog because I was already incrementally writing it for F-Droid release notes and thought it may interest some people (this in-app changelog is automatically built by reusing all the release notes). But in my opinion it doesn't worth the effort to translate it.

Reply:
I would translate it if it was on Weblate, but it isn't crucial and it is better to have in English that if it were to go missing due to no translations.
I might translate the fastlane files, don't know why it isn't generated from those?